test/Base_Tests/src/Network/Enso_Cloud/Cloud_Data_Link_Spec.enso

```diff
@@ -79,7 +79,7 @@ add_specs suite_builder setup:Cloud_Tests_Setup =
         group_builder.specify "should be able to read a local datalink to an Enso File" <|
             my_target_enso_file = "Hello Enso Cloud File!".write (test_root.get / "my-file.txt") . should_succeed
             datalink = transform_datalink (enso_project.data / "datalinks" / "example-enso-file.datalink") content->
-                content.replace "enso://PLACEHOLDER_ORG_NAME/PLACEHOLDER_PATH" my_target_enso_file.path
+                content.replace "enso://Teams/PLACEHOLDER_TEAM_NAME/PLACEHOLDER_PATH" my_target_enso_file.path
```

Comment:
I don't know the context but in a multi-tenant setup it would seem we need something like org/team/path structure? Or is team=org (naming is confusing)?

Reply:
I think Team is +- equivalent to Org, but that is a question to @PabloBuchu / @jdunkerley, the path structure refactor was done in https://github.com/enso-org/cloud-v2/pull/1344. This change only fixes the test that was forgotten to be adapted to the changed structure.